lms/static/scripts/frontend_apps/components/URLPicker.tsx

```diff
@@ -47,10 +47,12 @@ export default function URLPicker({
           setError('Please use a URL that starts with "http" or "https"');
         }
       } else if (isYouTubeURL(rawInputValue)) {
+        // TODO If YouTube is enabled, take the user to the right content picker instead of displaying an error
+        //      instructing how to do it manually
         setError(
           youtubeEnabled
             ? 'To annotate a video, go back and choose the YouTube option.'
-            : 'Annotating YouTube videos is not yet supported. This feature is coming soon.'
+            : 'Annotating YouTube videos is not supported.'
```

Comment:
```suggestion
            : 'Annotating YouTube videos is not enabled.'
```

"not supported" implies the capability doesn't exist, as opposed to being disabled.

Reply:
"Not enabled" kind of sounds the same as "not supported" or "we don't have this feature" to a user. Maybe we can say something that explains that it has been _disabled_ or _disallowed_ by this user's institution.

Reply:
I think this message needs someone less technical to think about it.

I went for "not supported" because it discloses less information. The user will think it cannot be done and that's it. However, should the user actually be informed it was actively disabled by someone? Who would usually be that someone, us or some institution admin?